Open Judge System/Workers/OJS.Workers.ExecutionStrategies/NodeJsPreprocessExecuteAndCheckExecutionStrategy.cs

```diff
@@ -183,16 +183,18 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
             // In NodeJS there is no compilation
             result.IsCompiledSuccessfully = true;
 
+            var executor = new RestrictedProcessExecutor();
+
             // Preprocess the user submission
```

Comment:
The comments may be mostly redundant, but in the base classes they are usefull to orient new people who may wish to work on the code.